### DISCUSSION TITLE
Using separate cross section and covariance ENDF6 files in the develop branch

### DISCUSSION THREAD

**Discussion by cdunn314** [luca-fiorito-11/sandy#323] [Q&A]
I would like to perturb the ENDF/B-VIII.0 Cl-35 cross sections, which do not have covariances in their ENDF6 file, but I would like to use the covariances from TENDL-2019 to create perturbations to the ENDF/B_VIII.0 cross sections. It seems this is readily achievable in the master branch of SANDY, but I was wondering if it is possible to do in the current develop branch? I am inclined to use the develop branch, as my code directly calls the `endf6.get_perturbations()` and `endf6.apply_perturbations()` so that I can ensure that the heatr NJOY module is run with local photon heating turned on. 

If this is not possible, would it bye possible to use the master branch to process separate cross section and covariances files and use the heatr module with local photon heating?

**Comment by luca-fiorito-11** [comment]
I would not recommend using the master branch, but rather v1.1, where we are doing active development.
Other users reported interest in having TENDL data for chlorine, see #319 .
PR #321, as soon as accepted, will give access to TENDL-2023. If interested, you might consider pushing something similar for TENDL-2019.

Alternatively, consider downloading the ENDF-6 file of interest locally, and then read it with `sandy.Endf6.from_file`.

**Reply by cdunn314** [reply]
Thank you! I had just downloaded the ENDF-6 files from TENDL-2019 previously, but that PR should streamline things in my code.

However, I still don't understand how to use a separate ENDF-6 file for covariances as for cross sections using v1.1. It didn't look like v1.1 had a `--cov` argument parser in the sampling.py script. How might I go about using one ENDF-6 file for the covariance and another for the cross sections using v1.1 or the develop branch?

**Comment by luca-fiorito-11** [comment]
See #319
